File: FrontEnd/Frontend_C#/SoftPac/SoftPacPersistance/DAO/TiposDeCambioDAO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using SoftPac.Model;

namespace SoftPac.Persistance.DAO
{
    public interface TiposDeCambioDAO
    {
        int Insertar(TiposDeCambioDTO tipoDeCambio);
        int Modificar(TiposDeCambioDTO tipoDeCambio);
        int Eliminar(TiposDeCambioDTO tipoDeCambio);
        TiposDeCambioDTO ObtenerPorId(int tipoDeCambioId);
        IList<TiposDeCambioDTO> ListarTodos();
    }
}

File: BackEnd/SoftPacPersistance/src/main/java/pe/edu/pucp/softpac/dao/TiposDeCambioDAO.java
package pe.edu.pucp.softpac.dao;

import pe.edu.pucp.softpac.model.TiposDeCambioDTO;

import java.util.ArrayList;
import java.util.List;

public interface TiposDeCambioDAO {
    public Integer insertar(TiposDeCambioDTO tipoDeCambio);

    public TiposDeCambioDTO obtenerPorId(Integer tipoDeCambioId);

    public List<TiposDeCambioDTO> listarTodos();

    public Integer modificar(TiposDeCambioDTO tipoDeCambio);

    public Integer eliminar(TiposDeCambioDTO tipoDeCambio);

}
